modules/odc-importing-a-codebase-from-git-to-create-an-application.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// applications/application-life-cycle-management/odc-creating-an-application-using-the-developer-perspective.adoc
+
+[id="odc-importing-a-codebase-from-git-to-create-an-application_{context}"]
+= Importing a codebase from Git to create an application
+
+The following tutorial walks you through the *Import from Git* option to create an application.
+
+Create, build, and deploy an application on {product-title} using an existing codebase in GitHub as follows:
+
+.Prerequisites
+
+* Ensure that you have link:https://docs.openshift.com/container-platform/latest/web-console/web-console.html[logged in to the web console].
+* Ensure that you have switched to the *Developer* perspective.
+//<link:url[label]>
+
+.Procedure
+
+. In the *Add* view, click *From Git* to see the *Import from git* form.
++
+image::odc_import_from_git.png[Import from Git]
+
+. In the *Git* section, enter the Git repository URL for the codebase you want to use to create an application. For this tutorial, enter the URL of this sample nodejs application `https://github.com/sclorg/nodejs-ex`.
+. Optionally, you can click *Show Advanced Git Options*  to add details such as:
+
+* *Git Reference* to point to code in a specific branch, tag, or commit to be used to build the application.
+* *Context Dir* to specify the subdirectory for the application source code, you want to use to build the application.
+* *Source Secret* to create a *Secret Name* with credentials for pulling your source code from a private repository.
+
++
+For this tutorial you do not need to use the advanced options.
+
+. In the *Builder* section, select the *Node.js*  builder image to see the details of the builder image. If required, you can change the version using the *Builder Image Version* drop-down.
```

Comment:
drop-down menu.

Reply:
Used drop-down lists instead, since they are part of a form and there are no menus.